test/swoosh/test_assertions_test.exs

```diff
@@ -292,6 +299,16 @@ defmodule Swoosh.TestAssertionsTest do
     end
   end
 
+  test "refute email sent with specific header" do
+    refute_email_sent(headers: unquote(%{"Revengers" => "Gather"}))
+  end
+
+  test "refute email sent with expected header" do
+    assert_raise ExUnit.AssertionError, fn ->
+      refute_email_sent(headers: unquote(%{"Avengers" => "Assemble"}))
+    end
+  end
+
```

Comment:
These tests may show that it still needs some work, since I couldn't get the tests working without the `unquote`

Reply:
Thanks. I pushed some refactoring on refute. Works now.